docs/source/tts/checkpoints.rst

```diff
@@ -79,7 +79,11 @@ There are multiple TTS tutorials provided in the directory of `tutorials/tts/ <h
 NGC TTS Models
 -----------------------------------
 
-Below summarizes a full list of available NeMo TTS models that have been released in `NGC NeMo Text to Speech Collection <https://catalog.ngc.nvidia.com/orgs/nvidia/collections/nemo_tts/entities>`_.
+This section summarizes a full list of available NeMo TTS models that have been released in `NGC NeMo Text to Speech Collection <https://catalog.ngc.nvidia.com/orgs/nvidia/collections/nemo_tts/entities>`_. You can download model checkpoints of your interest via either way below,
+
+* :code:`wget '<CHECKPOINT_URL_IN_THE_TABLE>'`
+* :code:`curl -LO '<CHECKPOINT_URL_IN_THE_TABLE>'`
```

Comment:
May also want to mention `from_pretrained(<model_tag>)` here.

Also, it would be nice to have that as a column in the table (but not strictly necessary--users can follow the link to view the versions in the worst case).

Reply:
moved model name to the 2nd column. 

Actually model name column includes `<model_tag>` as you mentioned. `from_pretrained` function has argument `model_name` instead of `model_tag`. So it would be better to keep the naming consistent.

Reply:
The latest screenshot is updated. Please have a look.